src/components/FormRegister.vue

```diff
@@ -1,41 +1,55 @@
 <template>
-  <form class="form-wrapper" @submit.prevent="onSubmit">
+  <form
+    class="form-wrapper"
+    @submit.prevent="onSubmit"
+  >
     <BaseInput
       type="email"
       label="email"
       placeholder="Enter your email address"
       v-model="email"
       @blur="isEmailTouched = true"
     />
-    <div>{{ emailError }}</div>
+    <div class="error-info">{{ emailError }}</div>
     <PasswordInput
       label="password"
       v-model="password"
       @blur="isPasswordTouched = true"
     />
-    <div>{{ passwordError }}</div>
+    <div class="error-info">{{ passwordError }}</div>
     <div class="form-wrapper__hints">
-        <p>At least 8 characters</p>
-        <p>At least one letter</p>
-        <p>At least one digit</p>
+      <p :class="[{red: !passwordLength && isPasswordTouched}, {green: passwordLength}]">At least 8 characters</p>
+      <p :class="[{red: !passwordLetter && isPasswordTouched}, {green: passwordLetter}]">At least one letter</p>
+      <p :class="[{red: !passwordDigit && isPasswordTouched}, {green: passwordDigit}]">At least one digit</p>
```

Comment:
What do you think about moving this `!passwordLength && isPasswordTouched` part to the computed property and then use it here? I think it could make this shorter
Or create a method
```
 isRed(param) {
  return  !param && isPasswordTouched;
 }
```
 and then use it here as:
  `red: isRed(passwordLength), , {green: passwordLength}`
   `red: isRed(passwordLetter), , {green: passwordLength}`
   `red: isRed(passwordDigit), , {green: passwordLength}`